Sources/MapboxMaps/Foundation/Camera/AnimationCompletion.swift

```diff
@@ -0,0 +1,3 @@
+import UIKit
+
+public typealias AnimationCompletion = (UIViewAnimatingPosition) -> Void
```

Comment:
Why does this get its own file?

Reply:
There used to be a delegate protocol declared in this file. The file was named after that delegate protocol. The delegate protocol has been deleted, so I renamed the file after the `typealias` instead.